File: Modul 4/Modul 4 Bahasa python/PRAK403-2410817310007-MuhammadNaufalAbdillah.py
mulai, henti =map(int,input(" ").split())

i,j=mulai,henti

if mulai<henti:
    while i<=henti and j>=mulai:
        print(f"{i} {j}", end=" ")
        
        if i<henti or j>mulai:
            print("-", end=" ")
        i +=1
        j -=1
elif i>j:
    while i>= henti and j<= mulai:
        print(f"{i} {j}", end=" ")

        if i>henti or j < mulai:
            print("-", end=" ")
        i -=1
        j +=1

File: Modul 4/Modul 4 Bahasa C/PRAK403-2410817310007-MuhammadNaufalAbdillah.c
#include <stdio.h>
int main(){
    int mulai, henti, i, j;
    scanf("%d %d", &mulai, &henti);

    for (i=mulai, j=henti; i<=henti&&j>=mulai; i++, j-- )
    {
    printf("%d %d",     i, j);
    if (i<henti||j>mulai)
        { printf(" - ");}
    }

    for (i=mulai, j=henti; i>=henti && j<=mulai; i--, j++ )
    {
       printf("%d %d", i, j);
       if (i>henti||j<mulai)
       {printf(" - ");}
       
    }
    return 0;
}